### PR TITLE
Some minor multisig updates

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

**Review by fmorency — COMMENTED** [review]
LGTM. Some minor clarification.
Thanks!

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:24]: How can one submit a multisig-transaction without having the `account-multisig-token`?

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:22]: Why not a `tstr`?

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:113]: Why change this to a `bstr`?

**Review by fmorency — COMMENTED** [review]
Some discussion points.

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (hansl) [reply attributes/network/9_account_features/cddl/1_account_multisig.cddl:24]: Urgh, yeah. This should be in the transaction history but not if you create a new transaction. I'll make it optional and specify.

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (hansl) [reply attributes/network/9_account_features/cddl/1_account_multisig.cddl:113]: See above. I'm open to discussion.

File: attributes/network/9_account_features/1_account_multisig.adoc
Comment (hansl) [reply attributes/network/9_account_features/1_account_multisig.adoc:57]: Nothing, and that might be a problem.

Comment (hansl) [reply attributes/network/9_account_features/1_account_multisig.adoc:57]: Will do this.

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (hansl) [reply attributes/network/9_account_features/cddl/1_account_multisig.cddl:22]: Let's have both a memo line (`tstr`) and a `metadata` line (`bstr`).

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:127]: `transaction-ledger-info` doesn't exist anywhere. Did you mean `ledger-transaction-info`? And if so, why change it from `account-multisig-transaction = ledger-transaction-info`?

**Review by fmorency — APPROVED** [review]
LGTM assuming you fix the minor comments.

You might also want to rephrase 

```
Additionally, `"owner"` is allowed to withdraw transactions, and update defaults.
```

located in the adoc file, following yesterday's discussion.

Thanks!

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:164]: This is an `Option` in `many-rs`.

**Review by fmorency — CHANGES_REQUESTED** [review]
Requesting clarification after `many-rs` and `many-framework` review.

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl
Comment (hansl) [reply attributes/network/9_account_features/cddl/1_account_multisig.cddl:127]: How did this pass CI then? We type check the CDDL...

Comment (hansl) [reply attributes/network/9_account_features/cddl/1_account_multisig.cddl:127]: Apparently, the CDDL check doesn't validate this... Urgh.

File: attributes/network/9_account_features/1_account_multisig.adoc
Comment (hansl) [reply attributes/network/9_account_features/1_account_multisig.adoc:57]: > If the sender is an account owner then it can change the defaults without others' approval.

This will be by design. I'll add a note that if you want you can set the account as its own owner. But the owner will always have all-powerful access to the account.

**Review by fmorency — APPROVED** [review]

**Pull request merged**
